test/instances/ezETH/EzETHHyperdrive.t.sol

```diff
@@ -0,0 +1,1081 @@
+        variableRate = variableRate.normalizeToRange(0.01e18, 2.5e18);
+        advanceTime(POSITION_DURATION_2_WEEKS, variableRate);
+
+        // Bob attempts to close his short with ETH as the target asset. This
+        // fails since ETH isn't supported as a withdrawal asset.
+        vm.stopPrank();
+        vm.startPrank(bob);
+        vm.expectRevert(IHyperdrive.UnsupportedToken.selector);
+        hyperdrive.closeShort(
+            maturityTime,
+            shortAmount,
+            0,
+            IHyperdrive.Options({
+                destination: bob,
+                asBase: true,
+                extraData: new bytes(0)
+            })
+        );
+    }
+
+    function test_close_short_with_ezeth(
+        uint256 shortAmount,
+        int256 variableRate
+    ) external {
+        // Accrue interest for a term to ensure that the share price is greater
+        // than one.
+        advanceTime(POSITION_DURATION_2_WEEKS, 0.05e18);
+
+        // Bob opens a short.
+        shortAmount = shortAmount.normalizeToRange(
+            2 * hyperdrive.getPoolConfig().minimumTransactionAmount,
+            HyperdriveUtils.calculateMaxShort(hyperdrive)
+        );
+        uint256 balanceBefore = bob.balance;
+        vm.deal(bob, shortAmount);
+        (uint256 maturityTime, uint256 basePaid) = openShort(bob, shortAmount);
+        vm.deal(bob, balanceBefore - basePaid);
+
+        // The term passes and interest accrues.
+        uint256 startingVaultSharePrice = hyperdrive
+            .getPoolInfo()
+            .vaultSharePrice;
+        variableRate = variableRate.normalizeToRange(0, 2.5e18);
+        advanceTime(POSITION_DURATION_2_WEEKS, variableRate);
+
+        // Get some balance information before closing the short.
+        (, , uint256 totalPooledEtherBefore) = RESTAKE_MANAGER.calculateTVLs();
+        uint256 totalSharesBefore = EZETH.totalSupply();
+        AccountBalances memory bobBalancesBefore = getAccountBalances(bob);
+        AccountBalances memory hyperdriveBalancesBefore = getAccountBalances(
+            address(hyperdrive)
+        );
+
+        // Bob closes his short with ezETH as the target asset. Bob's proceeds
+        // should be the variable interest that accrued on the shorted bonds.
+        uint256 expectedBaseProceeds = shortAmount.mulDivDown(
+            hyperdrive.getPoolInfo().vaultSharePrice - startingVaultSharePrice,
+            startingVaultSharePrice
+        );
+        uint256 shareProceeds = closeShort(
+            bob,
+            maturityTime,
+            shortAmount,
+            false
+        );
+        (, , uint256 totalPooledEther) = RESTAKE_MANAGER.calculateTVLs();
+        uint256 totalShares = EZETH.totalSupply();
+        uint256 baseProceeds = shareProceeds.mulDivDown(
+            totalPooledEther,
+            totalShares
+        );
+        // TODO: make sure this is OK.  There is a lot of rounding that happens in calculateTVLs
```

Comment:
We'll need to reevaluate all of the tolerances in this file once the conversion functions are updated. It would be good to comb through and reset these to the old values before another review pass.

Reply:
per the following comments about tolerance, resolving this.